backend/src/services/enhanced_guide_service.py

```
@@ -17,8 +17,40 @@
+    env_path = Path(__file__).parent.parent.parent.parent / '.env'
+    if env_path.exists():
+        with open(env_path) as f:
+            for line in f:
+                if line.strip() and not line.startswith('#'):
+                    key, value = line.strip().split('=', 1)
+                    os.environ[key] = value
+
+def get_api_key(key_name: str) -> str:
+    """Get API key from environment"""
+    return os.getenv(key_name, "")
+import logging
```

Ruff (0.13.1)

Redefinition of unused `logging` from line 9

Remove definition: `logging`

(F811)